tests/frontends/sdk/test_client.py

```
@@ -83,66 +83,88 @@ async def test_remote_node_delete(self):
+            # Create a node
+            await client.create_node("node-1", command="bash")
 
-        nodes = await client.list_nodes()
+            nodes = await client.list_nodes()
 
-        assert "node-1" in nodes
+            assert "node-1" in nodes
 
     @pytest.mark.asyncio
-    async def test_standalone_get_node(self):
+    async def test_standalone_get_node(self, tmp_path):
```

Ruff (0.14.10)

Unused method argument: `tmp_path`

(ARG002)